resources/helm/v2.1/gateways/istio-ingress/values.yaml

```diff
@@ -24,7 +24,11 @@ gateways:
       targetPort: 8443
       name: https
       protocol: TCP
-      # This is the port where sni routing happens
+    - port: 15012
```

Comment:
We should not be exposing this port over the gateway.  Our support for mesh expansion will automatically add this port.

Reply:
Removed [here](https://github.com/maistra/istio-operator/pull/737/commits/7283c4555e644786340c06d3081bc919251bf5c9). Thanks.